src/taipy/rest/api/resources/job.py

```diff
@@ -173,7 +173,7 @@ def post(self):
     def __create_job_from_schema(self, task_name: str) -> Optional[Job]:
         task_manager = TaskManager()
         try:
-            task = task_manager._get_or_create(self.fetch_config(task_name))
+            task = task_manager._bulk_get_or_create([self.fetch_config(task_name)])[0]
```

Comment:
I believe bulk_get_or_create returns a dictionary (config/entity). I believe `[0]` does not allow you to retrieve the task.

Reply:
@jrobinAV  I believe it is returning a list of tasks.

Reply:
The one you are showing is of DataManager. This is the signature of TaskManager. I am not sure if we should have inconsistent signature between managers.

```python
    def _bulk_get_or_create(
        cls,
        task_configs: List[TaskConfig],
        scenario_id: Optional[ScenarioId] = None,
        pipeline_id: Optional[PipelineId] = None,
    ) -> List[Task]:
```

Reply:
Oh my bad I was not looking at the right manager method !